### ISSUE TITLE
Setting assembly info for NetFramework doesn't update the version of the dlls produced.

### ISSUE THREAD

**Issue by cbfont01** [BMuuN/vsts-assemblyinfo-task#148]
I use the default fields but change the file version and the assembly version to 9.9.9.9. After this task, I copy the *.dll files to the staging directory before publishing them. When I download the published files, the version is unchanged.

**Comment by jay1891** [comment]
Hi, I notice the same issue.

**Comment by jay1891** [comment]
The following upgrade guide help, I think. But then I get multiple "error CS1056: Unexpected character ' ' "

https://github.com/BMuuN/vsts-assemblyinfo-task/wiki/Upgrade-v1-to-v2

**Comment by cbfont01** [comment]
I am currently using Assembly-Info-NetFramework@2 so the version isn't my problem. I haven't gotten any errors.

I haven't been able to get AssemblyInfo to work prior to the build step. After the build step, the Dlls are already built. My guess is that AssemblyInfo only modifies the AssemblyInfo.cs files.

As my build artifacts are mainly Dlls, this makes the extension useless.

**Comment by IanMercer** [comment]
@cbfont01  Maybe if you showed the output from the assembly info stamping step someone could help you. It probably isn't finding the files to update. It obviously cannot work after you've built the DLLs.

**Comment by cbfont01** [comment]
I thought the extension would find the Dll files and update their version after the build was created. I thought the 'Build.SourcesDirectory' was created by the build. Swapping the two steps fixes my problem. Thank you.

I had previously tried to swap the AssemblyInfo and the Build steps and it didn't work. I just looked at my history, and when I tried that I was pointing at the *.csproj files instead of the AssemblyInfo.cs.

For future reference, is this the best place to get help using a marketplace extension? I tried asking on SuperUser but the question got closed as off topic. Apparently an Extension does not count as Software.
